ksql-functional-tests/src/test/resources/query-validation-tests/session-windows.json

```diff
@@ -99,32 +99,12 @@
         {"topic": "test_topic", "key": 1, "value": "1,100,200", "timestamp": 40000}
       ],
       "outputs": [
-        {"topic": "S2", "key": 0, "value": "0,0", "timestamp": 0, "window": {"start": 0, "end": 30000, "type": "time"}},
-        {"topic": "S2", "key": 0, "value": null, "timestamp": 0, "window": {"start": 0, "end": 30000, "type": "time"}},
-        {"topic": "S2", "key": 0, "value": "0,5", "timestamp": 10000, "window": {"start": 0, "end": 40000, "type": "time"}},
-        {"topic": "S2", "key": 1, "value": "1,100", "timestamp": 10000, "window": {"start": 10000, "end": 40000, "type": "time"}},
-        {"topic": "S2", "key": 1, "value": null, "timestamp": 10000, "window": {"start": 10000, "end": 40000, "type": "time"}},
-        {"topic": "S2", "key": 1, "value": "1,200", "timestamp": 40000, "window": {"start": 10000, "end": 70000, "type": "time"}}
-      ]
-    },
-    {
-      "name": "import session stream and group by",
-      "statements": [
-        "CREATE STREAM TEST (ID bigint, VALUE bigint) WITH (kafka_topic='test_topic', value_format='DELIMITED', WINDOW_TYPE='SESSION');",
-        "CREATE TABLE S2 as SELECT ROWKEY as Id, count(*) FROM test GROUP BY ROWKEY;"
-      ],
-      "inputs": [
-        {"topic": "test_topic", "key": 0, "value": "0,0", "timestamp": 0, "window": {"start": 0, "end": 0, "type": "session"}},
-        {"topic": "test_topic", "key": 0, "value": null, "timestamp": 0, "window": {"start": 0, "end": 0, "type": "session"}},
-        {"topic": "test_topic", "key": 0, "value": "0,5", "timestamp": 10000, "window": {"start": 0, "end": 10000, "type": "session"}},
-        {"topic": "test_topic", "key": 1, "value": "1,100", "timestamp": 10000, "window": {"start": 10000, "end": 10000, "type": "session"}},
-        {"topic": "test_topic", "key": 1, "value": "1,200", "timestamp": 10000, "window": {"start": 10000, "end": 10000, "type": "session"}}
-      ],
-      "outputs": [
-        {"topic": "S2", "key": "0 : Window{start=0 end=0}", "value": "0 : Window{start=0 end=0},1", "timestamp": 0},
-        {"topic": "S2", "key": "0 : Window{start=0 end=10000}", "value": "0 : Window{start=0 end=10000},1", "timestamp": 10000},
-        {"topic": "S2", "key": "1 : Window{start=10000 end=10000}", "value": "1 : Window{start=10000 end=10000},1", "timestamp": 10000},
-        {"topic": "S2", "key": "1 : Window{start=10000 end=10000}", "value": "1 : Window{start=10000 end=10000},2", "timestamp": 10000}
+        {"topic": "S2", "key": 0, "value": "0,0", "timestamp": 0, "window": {"start": 0, "end": 9223372036854775807, "type": "time"}},
```

Comment:
The end of the window should be Long.MAX_VALUE since in legacy mode we read this as a time window with no size.